tests/sqlparser_postgres.rs

```diff
@@ -2733,6 +2733,39 @@ fn parse_create_brin() {
     }
 }
 
+#[test]
+fn parse_create_table_with_inherits() {
+    let single_inheritance_sql =
+        "CREATE TABLE child_table (child_column INT) INHERITS (public.parent_table)";
+    match pg().verified_stmt(single_inheritance_sql) {
+        Statement::CreateTable(CreateTable {
+            inherits: Some(inherits),
+            ..
+        }) => {
+            assert_eq_vec(&["public", "parent_table"], &inherits[0].0);
+        }
+        _ => unreachable!(),
+    }
+
+    let double_inheritance_sql = "CREATE TABLE child_table (child_column INT) INHERITS (public.parent_table, pg_catalog.pg_settings)";
+    match pg().verified_stmt(double_inheritance_sql) {
+        Statement::CreateTable(CreateTable {
+            inherits: Some(inherits),
+            ..
+        }) => {
+            assert_eq_vec(&["public", "parent_table"], &inherits[0].0);
+            assert_eq_vec(&["pg_catalog", "pg_settings"], &inherits[1].0);
+        }
+        _ => unreachable!(),
+    }
+}
+
+#[test]
+#[should_panic]
+fn parse_create_table_with_empty_inherits_fails() {
+    pg().verified_stmt("CREATE TABLE child_table (child_column INT) INHERITS ()");
```

Comment:
in place of the `should_panic`, we can rewrite the test case [in this form](https://github.com/apache/datafusion-sqlparser-rs/blob/165164bfc2d19fc61b3caec3fec2265c9173766a/tests/sqlparser_postgres.rs#L307-L310)
It would let us include the test case in the existing test function `parse_create_table_with_inherits` to help with grouping related test cases

Reply:
Refactored test as described!